infrastructure/validators/offboarding/stop-validating.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Stop Validating
+description: Instructions on how to stop validating, unbonding your tokens, purging validator keys, and chilling your Polkadot validator.
+---
+
+If you wish to remain a validator or nominator (for example, you're only stopping for planned downtime or server maintenance), submitting the `chill` extrinsic in the `staking` pallet should suffice. It is
+only if you wish to unbond funds or reap an account that you should continue with the following.
+
+To ensure a smooth stop to validation, make sure you should do the following actions:
+
+- Chill your validator
+- Purge validator session keys
+- Unbond your tokens
+
+These can all be done with [Polkadot.js](https://polkadot.js.org/apps){target=_blank} interface or with extrinsics.
```

Comment:
Instead of saying `with extrinsics` we could mention something like this:
```suggestion
These can all be done with [Polkadot.js Apps](https://polkadot.js.org/apps){target=\_blank} UI or by using various [libraries](https://wiki.polkadot.network/docs/build-tools-index#libraries){target=\_blank} such as [Polkadot.js API](https://polkadot.js.org/docs/){target=\_blank}.
```

Reply:
Agreed, will just change to PAPI since PJS is being phased out